src/IW/App/Error.hs

```diff
@@ -98,13 +101,15 @@ data AppErrorType
     -}
     | MissingHeader HeaderName
     {- | An authentication header that was required was provided but not in a
-    format that the server can understand
+    format that the server can understand.
     -}
     | HeaderDecodeError Text
-    -- | Data base specific errors
+    {- | Data base specific errors. -}
     | DbError Text
-    -- | Data base named parameters errors.
+    {- | Data base named parameters errors. -}
     | DbNamedError PgNamedError
+    {- | An error occurred while attempting to parse a @.cabal@ file. -}
+    | CabalParseError (Maybe Version, [CabalPError])
```

Comment:
I would prefer to put ` (Maybe Version, [CabalPError])` in a separate data type, like `ParseErrorInfo` for readability. This particular tuple doesn't look readable enough.

Reply:
@chshersh Okay got it.

Reply:
@chshersh I chose to name it `CabalParseErrorInfo` but it's kinda long. I just thought that `ParseErrorInfo` was kind of vague.

Reply:
@chshersh Changed to `CabalErrorInfo`